index.html

```diff
@@ -46,4 +46,113 @@ <h3 class="presentation">I'm a Software Developer</h3>
             class="social-icon"></img></a></li>
     </ul>
   </section>
+  <section class="works">
+    <h2 class="recent_works">My Recent Works</h2>
+    <hr class="recent_works_break">
+    <div class="main_project">
+      <img class="project_image" src="./images/content/project_main.svg" alt="project image">
+      <h3 class="work_title_main">Multi-Post Stories</h3>
+      <p class="work_description_main">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+      <ul class="tools_list_main">
+        <li class="tool_main">CSS</li>
+        <li class="tool_main">HTML</li>
+        <li class="tool_main">Bootstrap</li>
+        <li class="tool_main">Ruby</li>
+      </ul>
+      <a href="#" class="see_project_main">See Project</a>
+    </div>
+    <div class="project_card card1">
+        <h3 class="card_title">Professional Art<br> Printing Data</h3>
+        <p class="card_description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <ul class="tools_list_card">
+          <li class="tool_card">HTML</li>
+          <li class="tool_card">Bootstrap</li>
+          <li class="tool_card">Ruby</li>
+        </ul>
+        <a href="#" class="see_project_card">See Project</a>
+    </div>
```

Comment:
- You have done really tremendous work with this project :clap: :clap: :muscle: 

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/252012601-c5f1b5a7-a34a-4d52-84d7-461f47c82958.png)|![image](https://user-images.githubusercontent.com/99715640/252011801-c112a726-6d7a-43a2-a460-a1c27dc5a71d.png)|

 - [ ] How about reducing the space between the first and the second project cards to match Figma's design?
 - [ ] Also, I think we are missing some color attributes in the first card. The project title and the paragraph is not as we have in the Figma design.
 - [ ] Lastly, please note the overflow on the `y-axis`. It's not good for user experience